characterisation/exceptions/HO200116.test.ts

```diff
@@ -0,0 +1,37 @@
+import World from "../../utils/world"
+import { processPhase2Message } from "../helpers/processMessage"
+import { asnPath } from "../helpers/errorPaths"
+import generatePhase2Message from "../helpers/generatePhase2Message"
+import MessageType from "../types/MessageType"
+
+describe.ifPhase2("HO200116", () => {
+  afterAll(async () => {
+    await new World({}).db.closeConnection()
+  })
+
+  it.each([MessageType.ANNOTATED_HEARING_OUTCOME, MessageType.PNC_UPDATE_DATASET])(
+    "creates a HO200116 exception when there are more than 100 offences",
```

Comment:
```suggestion
    "creates a HO200116 exception for %s when there are more than 100 offences",
```

So each test has a unique test description.